x/nft/go.mod

```diff
@@ -1,6 +1,6 @@
 module cosmossdk.io/x/nft
 
-go 1.21
+go 1.21.0
```

Comment:
Should we revert to 1.21? I believe a go mod tidy changed them, because of x/pool specifies 1.21.0.
I think minor should matter here.